src/components/ResourceList/components/CheckableButton/CheckableButton.tsx

```diff
@@ -24,7 +30,28 @@ export function CheckableButton({
   plain,
   measuring,
   disabled,
+  smallScreen,
 }: CheckableButtonProps) {
+  const checkBoxRef = useRef<CheckboxHandles>(null);
+
+  const {registerCheckableButtons} = React.useContext(ResourceListContext);
+
+  let currentKey: CheckableButtonKey = 'plain';
+
+  if (plain) {
```

Comment:
Curious what you think about a switch statement here?

```
case plain:
  break;
case smallScreen:
  currentKey = 'bulkSm';
  break;
default:
  currentKey = 'bulkLg';
```

or if there's a condition that lets us remove `currentKey = 'plain';` or default `currentKey` to undefined. Just some written thoughts, don't let this block you from 🐑 🇮🇹

Reply:
in this case, we're switching on 2 different props. I can simply by defaulting to the `else` instead.